doc/api/pypeit.rst

```diff
@@ -24,10 +24,10 @@ Submodules
    pypeit.biasframe
    pypeit.bitmask
    pypeit.calibrations
-   pypeit.check_requirements
    pypeit.coadd1d
    pypeit.coadd2d
    pypeit.compiler_version
+   pypeit.compiler_version
```

Comment:
I'm not sure why this is in the file twice.  This file should be automatically generated by the `update_docs` script.  If you didn't edit this by hand, I can check why this is occurring twice.

Reply:
i did not edit any of the api docs by hand.

Reply:
Weird.  I just re-ran `update_docs`, and this line wasn't repeated.  I think it's fine, and we can fix it in `develop` if need be.